minor corrections (#19)
JuliaReach/SetPropagation-FEM-JuliaCon21: paper/paper.tex

\emph{Set-Based Solid Dynamics.} %
%

> When uncertainty is present, the initial displacements $\mathbf{x}(0)$ and the initial velocities velocities $\mathbf{x}'(0)$ can be considered as part of corresponding feasible states sets $\mathcal{X}_0$ and  $\mathcal{V}_0$, respectively. %

Reviewer: ```suggestion
When uncertainty is present, the initial displacements $\mathbf{x}(0)$ and the initial velocities $\mathbf{x}'(0)$ belong to the feasible sets $\mathcal{X}_0$ and  $\mathcal{V}_0$, respectively. %
```
